api/schema/tag_to_label_resource.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Illumio, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var (
+	tagToLabelResource = Resource{
+		TypeName: "tag_to_label",
+		Schema: resource_schema.Schema{
+			Version:             1,
+			MarkdownDescription: "Maps AWS account tags to CloudSecure labels.",
+			Attributes: map[string]resource_schema.Attribute{
+				IDFieldName: idAttribute,
+				"cloud_tags": resource_schema.ListAttribute{
```

Comment:
There is no ambiguity that these are "cloud" tags. In the `aws` provider's resources, this attribute is simply named `tags`, e.g., [in the `aws_instance` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#tags-3). Use the same name for consistency.
```suggestion
				"tags": resource_schema.ListAttribute{
```

Reply:
I'd actually rather split this into two attributes, cf. https://github.com/illumio/terraform-provider-illumio-cloudsecure/pull/116#discussion_r1911123532